config/400-config-experimental-features.yaml

```diff
@@ -0,0 +1 @@
+core/configmaps/experimental-features.yaml
```

Comment:
Can we call this just `features.yaml`, to be consistent with Knative Serving? A feature can be Alpha, ie experimental. A feature can also be stable and still be disabled if a vendor doesn't want it for X reason.

Reply:
I added a comment in the doc about that